spec/models/access_online_links_spec.rb

```diff
@@ -2,7 +2,7 @@
 
 require 'rails_helper'
 
-RSpec.describe AccessOnlineLinks do
+RSpec.describe ExternalLinks do
```

Comment:
Maybe we should rename this file to `external_links_spec.rb` to match the new name of the spec?

Reply:
oh yes! totally missed this one :) good catch :)

Reply:
ok just changed with the latest commit